app/src/main/java/com/bottlerocketstudios/brarchitecture/ui/DbViewModel.kt

```diff
@@ -0,0 +1,20 @@
+package com.bottlerocketstudios.brarchitecture.ui
+
+import androidx.lifecycle.LiveData
+import androidx.lifecycle.MutableLiveData
+import timber.log.Timber
+
+/**
+ * [Marker interface](https://en.wikipedia.org/wiki/Marker_interface_pattern) designating that the denoted class is a Databinding ViewModel, the last VM of the traditional MVVM
+ * (Model-View-ViewModel) pattern. The implementor usually takes in a data class with values that represent current state and provides databinding "observable" wrappers around the formatted data.
+ *
+ * See https://developer.android.com/topic/libraries/data-binding/observability
+ */
+interface DbViewModel {
```

Comment:
As we talked in JediLabs, another name for "DB". But not a show stopper.

Reply:
I wasn't there in JediLabs but Db seems self-explanatory.

Reply:
@harishrpatel @Josh-Gebbeken I'll stick with this for now. Feel free to propose another name in a future issue/PR 😄